nullaway/src/main/java/com/uber/nullaway/ErrorProneCLIFlagsConfig.java

```diff
@@ -152,6 +154,8 @@
             flags, FL_CLASS_ANNOTATIONS_TO_EXCLUDE, DEFAULT_CLASS_ANNOTATIONS_TO_EXCLUDE);
     initializerAnnotations =
         getFlagStringSet(flags, FL_INITIALIZER_ANNOT, DEFAULT_INITIALIZER_ANNOT);
+    customNullableAnnotation = flags.get(FL_NULLABLE_ANNOT).orElse("javax.annotation.Nullable");
```

Comment:
We already support `javax.annotation.Nullable` (and anything with simple name `@Nullable` natively), so this default is redundant. Also, users might want to specify more than one custom nullability annotation at the time in a large repo. 

My suggestion here would be to model **both** of these new options after `CustomInitializerAnnotations`, using `getFlagStringSet`, but use an empty immutable set (`ImmutableSet.of()` without arguments) as the default.

Reply:
1c56af6a5e5423c982c620cbd2476bdee23ee212